chatkit-core/src/main/kotlin/com/pusher/chatkit/CurrentUser.kt

```diff
@@ -428,9 +440,13 @@ class CurrentUser(
             makeCallback({ syncCurrentUser.usersForRoom(roomId) }, callback)
 
     /**
-     * Enables push notifications for the current user. The [callback] will be called with a [Result]
-     * when the operation is complete. If the operation was successful you will receive a [Unit],
-     * or an [Error] informing you of what went wrong.
+     * Enables push notifications for the current user. You can call this everytime a user connects successfully to Chatkit,
+     * a user can receive push notifications on up to 10 Android devices. However you should disable
+     * push notifications if you detect the user in a signed out state to ensure they don't accidentally receive further
+     * push notifications as per the [information here][https://pusher.com/docs/chatkit/guides/push_notifications/setup-android#disabling-push-notifications])
+     * To disable push notifications for the current user call `chatManager.disablePushNotifications {  }`
+     * The [callback] will be called with a [Result] when the operation is complete.
+     * If the operation was successful there is no value returned, or an [Error] informing you of what went wrong.
```

Comment:
This should probably be broken up in to paragraphs

Reply:
I've split this into 3 parts, as I think that keeps similar things together:

```
/**
     * Enables push notifications for the current user.
     *
     * You can call this every time a user connects successfully to Chatkit,
     * a user can receive push notifications on up to 10 Android devices. However you should disable
     * push notifications if you detect the user in a signed out state to ensure they don't accidentally receive further
     * push notifications. To disable push notifications for the current user call `chatManager.disablePushNotifications {  }`
     * More information on this is available in our
     * [docs][https://pusher.com/docs/chatkit/guides/push_notifications/setup-android#disabling-push-notifications])
     *
     * The [callback] will be called with a [Result] when the operation is complete.
     * If the operation was successful there is no value returned, or an [Error] informing you of what went wrong.
     */
```

Reply:
:+1: